src/data/search.tsx

```diff
@@ -67,7 +87,7 @@ const searchableCases = indexCases(data.cases);
 const searchableTopics = indexTopics(data.topics);
 
 export function searchContent(needle: string, contentType: string) {
```

Comment:
This method should guard against getting an empty search string, and return an empty array. Otherwise, it returns ALL results from the lunr.js index.